spec/features/admin/resources_integration_spec.rb

```diff
@@ -125,4 +125,39 @@
       expect(page).to have_content("Succesfully removed")
     end
   end
+
+  context "with event that acts_as_taggable" do
+    around do |example|
+      Event.class_eval { acts_as_taggable }
+      example.run
+      # Unload the monkey patch
+      Object.send(:remove_const, :Event)
+      load "spec/dummy/app/models/event.rb"
```

Comment:
No. I actually like it even.   
I just mentioned it, because the reason for the earlier failing build has been that I did not unload the monkey patch.